packages/packages_linux.go

```diff
@@ -142,7 +141,6 @@ func GetInstalledPackages(ctx context.Context) (Packages, error) {
 		if err != nil {
 			msg := fmt.Sprintf("error listing installed pip packages: %v", err)
 			clog.Debugf(ctx, "Error: %s", msg)
```

Comment:
If these errors don't count toward the success/failure of this function, should we raise their log level? On one hand, we don't want to be too verbose for common issues. On the other, silently failing like this can be really confusing. I almost wonder if we'd want a `logOnce` function to avoid log spew.

Reply:
We specifically dont log pip and gem errors because they are very prone to issues, I just missed these functions when I removed the errors.